src/icons/fileIcons.ts

```diff
@@ -1000,27 +1025,56 @@ export const fileIcons: FileIcons = {
     },
     {
       name: 'contributing',
-      fileNames: ['contributing.md'],
+      fileNames: [
+        'contributing.md',
+        'contributing.rst',
+        'contributing.txt',
+        'contributing',
+      ],
+    },
+    { name: 'readme',
+      fileNames: [
+        'readme.md',
+        'readme.rst',
+        'readme.txt',
+        'readme',
+      ],
     },
-    { name: 'readme', fileNames: ['readme.md', 'readme.txt', 'readme'] },
     {
       name: 'changelog',
       fileNames: [
         'changelog',
         'changelog.md',
+        'changelog.rst',
         'changelog.txt',
         'changes',
         'changes.md',
+        'changes.rst',
         'changes.txt',
+        'changes',
```

Comment:
`changes` is already  defined in line 1050. Please remove it to get the pipeline running again:

![image](https://user-images.githubusercontent.com/12248527/160332611-16e17a3c-f42b-4a11-a7b9-df17cb2781c2.png)

Reply:
i have no clue how that slipped by, just fixed it